### PR TITLE
New content for span events.

### PR DESCRIPTION
**Pull request by rhetoric101**
This is my initial draft for span events. This is pretty light on content, so I am asking for an SME review.

### CODE REVIEW

**Comment by rhetoric101** [comment]
@barbnewrelic, could you review this? I just flipped it back and forth from review-to-draft-to-review to get a reviewer and you were automagically assigned.

File: src/content/docs/integrations/open-source-telemetry-integrations/opentelemetry/opentelemetry-legacy-new-relic-exporters.mdx [hunk 1]
```diff
@@ -36,7 +36,7 @@ If you still need information about the New Relic Exporter, see the [opentelemet
 
 The [OpenTelemetry Collector with New Relic Exporter](https://github.com/newrelic/newrelic-opentelemetry-examples/tree/main/collector/nr-exporter-docker) example demonstrates what such a setup might look like.
 
-## Endpoint Configuraion for New Relic exporters [#h2-change-endpoints]
+## Endpoint Configuration for New Relic exporters [#h2-change-endpoints]
```
Comment (barbnewrelic) [inline src/content/docs/integrations/open-source-telemetry-integrations/opentelemetry/opentelemetry-legacy-new-relic-exporters.mdx:39]: Configuration s/b configuration

File: src/content/docs/integrations/open-source-telemetry-integrations/opentelemetry/view-your-opentelemetry-data-new-relic.mdx [hunk 2]
```diff
@@ -13,34 +13,64 @@ After you import OpenTelemetry data into New Relic, you can use a variety of too
...
+1. Go to [one.newrelic.com](https://one.newrelic.com).
 2. In the left sidebar, click **Services - OpenTelemetry**:
 
     ![Screen capture showing how to filter for OpenTelemetry services.](./images/explorer_otel_services.png "Filter for OpenTelemetry to see only those entities.")
 
-3. Click the service you want to know more about.
+3. In the center pane, click the service you want to know more about, or find your service by entering the name in the top filter bar.
 
-If you need help understanding the data, see the explanations which follow.
+If you need help understanding how we display your data, see the left-pane options below.
```
Comment (barbnewrelic) [inline src/content/docs/integrations/open-source-telemetry-integrations/opentelemetry/view-your-opentelemetry-data-new-relic.mdx:25]: would a link enclosing "left-pane options" be more helpful than "below" ? I'm not sure how far down this document I need to skim

File: src/content/docs/integrations/open-source-telemetry-integrations/opentelemetry/view-your-opentelemetry-data-new-relic.mdx [hunk 3]
```diff
@@ -13,34 +13,64 @@ After you import OpenTelemetry data into New Relic, you can use a variety of too
...
 
 ### Summary page [#summary]
 
-The opening page of the explorer is the **Summary** page listing various golden signals about your entity. Golden signals are key monitoring details such as response time, throughput, and error rate. By using this information, you can quickly decide if you need to dig deeper.
+After you click on a service in the **Explorer** tab, you see the **Summary** page listing various golden signals about your entity. Golden signals are key monitoring details such as response time, throughput, and error rate. By using this information, you can quickly decide if you need to dig deeper.
```
Comment (barbnewrelic) [inline src/content/docs/integrations/open-source-telemetry-integrations/opentelemetry/view-your-opentelemetry-data-new-relic.mdx:29]: I really love how you anticipated my question about what on earth is a golden signal?!

File: src/content/docs/integrations/open-source-telemetry-integrations/opentelemetry/view-your-opentelemetry-data-new-relic.mdx [hunk 4]
```diff
@@ -13,34 +13,64 @@ After you import OpenTelemetry data into New Relic, you can use a variety of too
...
+#### View span events
+
+If you send span events as described by the [OpenTelemetry specification](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/api.md#add-events), you can view them in the New Relic UI. Span events have two general types:
+  * Exceptions 
+  * Non-exceptions (for example, logs)
+
+If you have span events, links for these appear in the right pane:
+
+1. Click on a span in the waterfall view.
+2. In the right pane, click **View span events**, or you can expand **Error details** and click the link at the bottom of the errors.
```
Comment (barbnewrelic) [inline src/content/docs/integrations/open-source-telemetry-integrations/opentelemetry/view-your-opentelemetry-data-new-relic.mdx:64]: maybe consider changing "or you can" to just "or expand" to keep it sounding like a procedure

File: src/content/docs/integrations/open-source-telemetry-integrations/opentelemetry/view-your-opentelemetry-data-new-relic.mdx [hunk 5]
```diff
@@ -13,34 +13,64 @@ After you import OpenTelemetry data into New Relic, you can use a variety of too
...
+
+  ![Screenshot showing the right pane showing the two links for span events](./images/view-span-events.png "Screenshot showing the right pane showing the two links for span events")
+
+3. When you're in span events and only want to view exceptions, slide the toggle **Only show exceptions**.
```
Comment (barbnewrelic) [inline src/content/docs/integrations/open-source-telemetry-integrations/opentelemetry/view-your-opentelemetry-data-new-relic.mdx:68]: I usually just write "toggle" rather than "slide the toggle" but not insisting you must change this! :-)

File: src/content/docs/integrations/open-source-telemetry-integrations/opentelemetry/view-your-opentelemetry-data-new-relic.mdx [hunk 6]
```diff
@@ -13,34 +13,64 @@ After you import OpenTelemetry data into New Relic, you can use a variety of too
...
+
+![Screenshot showing span events and how you can filter just for exceptions](./images/span-events-exceptions.png "Screenshot showing span events and how you can filter just for exceptions")
+            
 <Callout variant="tip">
-  If you prefer to search traces across all New Relic accounts in your organization, you can go outside explorer: **[one.newrelic.com](https://one.newrelic.com) > Apps > Favorites > Distributed tracing.**
+OpenTelemetry exceptions handled by the app/service are displayed independently of span error status and are not necessarily associated with a span error status.
```
Comment (barbnewrelic) [inline src/content/docs/integrations/open-source-telemetry-integrations/opentelemetry/view-your-opentelemetry-data-new-relic.mdx:73]: maybe consider breaking this long sentence into its two thoughts. maybe reword to "...independently of span error status. These exceptions are not necessarily..."

**Review by barbnewrelic — APPROVED** [review]
Minor suggestions, take 'em or leave 'em! I think this is really succinct but provides a great workflow through how to use the UI effectively. After you make any changes you decide to do, go ahead and use your admin privileges and merge. No second review required!
PS: I never could get this to build in localhost but it looked just fine in the Gatsby preview.

File: src/content/docs/integrations/open-source-telemetry-integrations/opentelemetry/opentelemetry-legacy-new-relic-exporters.mdx (hunk 1 above)
Comment (rhetoric101) [reply src/content/docs/integrations/open-source-telemetry-integrations/opentelemetry/opentelemetry-legacy-new-relic-exporters.mdx:39]: @barbnewrelic , I was so focused on fixing the typo that I missed the capitalization issue! Thanks!

File: src/content/docs/integrations/open-source-telemetry-integrations/opentelemetry/view-your-opentelemetry-data-new-relic.mdx (hunk 2 above)
Comment (rhetoric101) [reply src/content/docs/integrations/open-source-telemetry-integrations/opentelemetry/view-your-opentelemetry-data-new-relic.mdx:25]: @barb, the options are immediately following, so I changed it to this: "...see the following descriptions of the left-pane options in the UI:"

File: src/content/docs/integrations/open-source-telemetry-integrations/opentelemetry/view-your-opentelemetry-data-new-relic.mdx (hunk 4 above)
Comment (rhetoric101) [reply src/content/docs/integrations/open-source-telemetry-integrations/opentelemetry/view-your-opentelemetry-data-new-relic.mdx:64]: @barbnewrelic, fixed it!

File: src/content/docs/integrations/open-source-telemetry-integrations/opentelemetry/view-your-opentelemetry-data-new-relic.mdx (hunk 5 above)
Comment (rhetoric101) [reply src/content/docs/integrations/open-source-telemetry-integrations/opentelemetry/view-your-opentelemetry-data-new-relic.mdx:68]: @barbnewrelic , I went with "toggle" as a verb. Thanks!

File: src/content/docs/integrations/open-source-telemetry-integrations/opentelemetry/view-your-opentelemetry-data-new-relic.mdx (hunk 6 above)
Comment (rhetoric101) [reply src/content/docs/integrations/open-source-telemetry-integrations/opentelemetry/view-your-opentelemetry-data-new-relic.mdx:73]: @barbnewrelic, great suggestion!
